client/src/components/Verejne/GoogleMap/index.js

```diff
@@ -43,6 +55,11 @@ const getClusters = (mapOptions: MapOptions, entities: Array<Entity>): Array<Clu
   return clusters(mapOptions)
 }
 
+const getClusterTooltip = (entity): string => {
+  const sorted = reverse(sortBy(entity.points, ['size']))
+  return sorted[0].name
+}
+
 const createClusters = (mapOptions: MapOptions, entities: Array<Entity>): Array<MapCluster> => {
```

Comment:
I probably missed earlier (or ignored because redux wasn't in play yet) but this belongs to a selector